klib/src/platform/windows/windows.def

```diff
@@ -1,3 +1,5 @@
 package = platform.windows
-headers = windows.h
-compilerOpts = -DUNICODE -Wno-incompatible-pointer-types -Wno-deprecated-declarations
\ No newline at end of file
+headers = windows.h commctrl.h dwmapi.h shlobj.h shlwapi.h shobjidl.h \
+    urlmon.h usp10.h uxtheme.h wininet.h vfw.h
+compilerOpts = -DUNICODE -DWINVER=0x0A00 -D_WIN32_WINNT=0x0A00 -DWINAPI_FAMILY=3 \
```

Comment:
0x0A00 is Windows 10, maybe we need separate versions of def files for different windows APIs. Maybe 0x0601 (Win 7) is somewhat more universal common ground. Also we need to discuss separation of declarations between different libraries, as Windows API is not entirely monolithic.

Reply:
Downgraded ro 0x601
About separation - this is still not complete Win32 API, mostly GUI part.
Dont't know how to split it...

Reply:
Maybe ```urlmon.h``` and ```wininet.h``` can be separated into WinINet module.

Reply:
For GUI part of APIs - main missed part is gdiplus.h - it is c++ library, but many GUI functionalities cannot be implemented without it. :(